ground/gcs/src/plugins/uavobjects/uavdataobject.h

```diff
@@ -37,7 +37,7 @@
 class UAVOBJECTS_EXPORT UAVDataObject : public UAVObject
 {
     Q_OBJECT
-
+    Q_PROPERTY(bool isPresent READ getIsPresentOnHardware WRITE setIsPresentOnHardware NOTIFY presentOnHardwareChanged)
```

Comment:
We should have exactly the same name as the c++ code in all our QML stuff or things will get messy. (i.e. `isPresentOnHardware`.